app/services/pii/fingerprinter.rb

```diff
@@ -1,13 +1,29 @@
 module Pii
   class Fingerprinter
-    def self.fingerprint(text)
+    def self.fingerprint(text, key = Figaro.env.hmac_fingerprinter_key)
       digest = OpenSSL::Digest::SHA256.new
-      key = Figaro.env.hmac_fingerprinter_key
       OpenSSL::HMAC.hexdigest(digest, key, text)
     end
 
     def self.verify(text, fingerprint)
+      verify_current(text, fingerprint) || verify_queue(text, fingerprint)
+    end
+
+    def self.verify_current(text, fingerprint)
       ActiveSupport::SecurityUtils.secure_compare(fingerprint, fingerprint(text))
     end
+
+    def self.verify_queue(text, fingerprint)
+      JSON.parse(Figaro.env.hmac_fingerprinter_key_queue || '[]').each do |key|
```

Comment:
this `JSON.parse(Figaro.env.hmac_fingerprinter_key_queue || '[]')` is repeated in a few places, should we make a helper method somewhere for it? ex `KeyRotator.old_keys` or something?